Seems to fail the same way on master. 

```bash
/build/source/GalaxyBudsClient/GalaxyBudsClient.csproj : error NU1101: Unable to find package Microsoft.NETCore.App.Host.linux-x64. No packages exist with this id in source(s): _nix
```

Looks like the dotnet flag was only set for x64 https://github.com/NixOS/nixpkgs/pull/365231/files#diff-0481f4c907609b2332cf110e6aad41041f189c410538e7f5613bc5fc9e3697a8R30

--- a/pkgs/by-name/ga/galaxy-buds-client/package.nix
+++ b/pkgs/by-name/ga/galaxy-buds-client/package.nix
@@ -14,20 +14,22 @@
 
 buildDotnetModule rec {
   pname = "galaxy-buds-client";
-  version = "5.1.0";
+  version = "5.1.1";
 
   src = fetchFromGitHub {
     owner = "ThePBone";
     repo = "GalaxyBudsClient";
     rev = version;
-    hash = "sha256-9m9H0T4rD6HIvb15h7+Q7SgLk0PkISkN8ojjh7nsiwA=";
+    hash = "sha256-Cie8dInNzqMS6k2XP2P3fwMxSc6AabZHiIc6vcA7VhM=";
   };
 
   projectFile = [ "GalaxyBudsClient/GalaxyBudsClient.csproj" ];
   nugetDeps = ./deps.json;
   dotnet-sdk = dotnetCorePackages.sdk_8_0;
   dotnet-runtime = dotnetCorePackages.runtime_8_0;
-  dotnetFlags = [ "-p:Runtimeidentifier=linux-x64" ];
+  dotnetFlags =
+    lib.optionals stdenv.hostPlatform.isx86_64 [ "-p:Runtimeidentifier=linux-x64" ]
+    ++ lib.optionals stdenv.hostPlatform.isAarch64 [ "-p:Runtimeidentifier=linux-arm64" ];
 
   nativeBuildInputs = [ makeWrapper copyDesktopItems ];
 

